pkg/aws/helpers.go

```diff
@@ -455,6 +455,10 @@ func GetPolicyARN(partition string, accountID string, name string, path string)
 	return getPolicyARN(partition, accountID, GetPolicyName(name), path)
 }
 
+func GetNormalPolicyARN(partition string, accountID string, name string, path string) string {
```

Comment:
Can we rename the other one to GetOcmPolicyArn and this to GetPolicyArn instead? :thinking: Not sure if it would make it clearer or not to be truthful

Reply:
Yeah this is a tough one to name. I think ill rename the other to `GetPolicyArnWithSuffix` and this one as `GetPolicyArn`